### PR TITLE
maven dependencies for java.xml.bind (jaxb) issue #103

### PR DESCRIPTION
**Pull request by ipolyzos**
add maven dependencies for the java.xml.bind (jaxb) module #103

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
         <snakeyaml.version>1.24</snakeyaml.version>
         <spock.version>1.3-groovy-2.5</spock.version>
         <jackson.version>2.9.9</jackson.version>
+        <jaxb-api>2.3.0</jaxb-api>
```
Comment (onderson) [inline pom.xml]: naming conv is by `.version`

Comment (onderson) [inline pom.xml]: and final version 2.3.1

Comment (ipolyzos) [reply pom.xml]: thanks, I  will proceed with the change in order to comply

Comment (ipolyzos) [reply pom.xml]: I have upgraded the dependency to v2.3.1

**Comment by ipolyzos** [comment]
the two changes requested have been committed

**Pull request merged**